lms/static/scripts/frontend_apps/services/test/vitalsource-test.js

```diff
@@ -0,0 +1,37 @@
+import { ApiError } from '../../utils/api';
```

Comment:
Not for this body of work: Should this be `APIError`?

Reply:
Yes, per our current conventions.